youtube_dl/extractor/dplay.py

```diff
@@ -5,40 +5,70 @@
 
 from .common import InfoExtractor
 from ..utils import int_or_none
-
+from ..compat import compat_urlparse
 
 class DPlayIE(InfoExtractor):
-    _VALID_URL = r'http://www\.dplay\.se/[^/]+/(?P<id>[^/?#]+)'
-
-    _TEST = {
-        'url': 'http://www.dplay.se/nugammalt-77-handelser-som-format-sverige/season-1-svensken-lar-sig-njuta-av-livet/',
```

Comment:
Strange, It was definitely working 15days ago.  I'll take a look at this later today.  It could be due to the setting of the cookie that I removed.